### ISSUE TITLE
Value with quote character is not displayed

### ISSUE THREAD

**Issue by Jaaap** [sa-si-dev/virtual-select#80]
With v1.0.19 and this snippet:
```javascript
VirtualSelect.init({
	ele: '#sample-select',
	multiple: true,
	showValueAsTags: true,
	dropboxWidth: "400px",
	enableSecureText: true,
	name: 'test',
	options: [
		{ label: 'Label 1', value: '1' },
		{ label: 'Label 2', value: 'a"b' },
	],
});
```
You can select the first option and it will be displayed as a value-tag.
1. If you select the second option, it is not displayed as a value-tag.
2. If you select both options and then deselect option one by clicking its clear-icon, both values will be deselected.

**Comment by sa-si-dev** [comment]
@Jaaap I will check and fix it in the future release.

**Comment by sa-si-dev** [comment]
@Jaaap this issue has been fixed in the release [v1.0.20](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.20)
